contracts/RenPool.sol

```diff
@@ -1,35 +1,43 @@
 pragma solidity ^0.8.0;
 
-import "OpenZeppelin/openzeppelin-contracts@4.0.0/contracts/token/ERC20/ERC20.sol";
+import "./RenToken.sol";
+
 
 contract RenPool {
-    ERC20 public renToken;
+    RenToken public renToken;
     address public owner; // This will be our address, in case we need to destroy the contract and refund everyone
     address public admin;
-    mapping(address => uint) public balances;
+    mapping(address => uint) public balanceOf;
     uint public totalPooled;
     bool public isLocked;
-    uint public ownerFee; // Percentage
-    uint public adminFee; // Percentage
+    uint public fee;
     uint public target;
```

Comment:
@JonathanStefanov be careful bc you are reverting some of my previous changes, `fee` variable for instance should now be `onwerFee` and `adminFee`, same applies to the name of the events. I'd suggest to start over from master and add your changes manually. Also, once you open a PR have a look at the changes you are introducion on github to see what things are being change by your PR